### ISSUE TITLE
Hii bro , I just have a small doubt regarding this radio player package on the app that I am working on can u please clarify it ?

### ISSUE THREAD

**Issue by RadioApp1** [JamsKs/Radio_App#1]

**Comment by JamsKs** [comment]
Sure what is your question

**Comment by RadioApp1** [comment]
The issue is am getting a default notification on top using this app with only play button . I also want to have next and previous buttons , how to dot that ? this is also preventing me to use custom notifications , so am stuck :(( plzzz help bro

**Comment by RadioApp1** [comment]
this is caused by the radio player package

**Comment by JamsKs** [comment]
Radio player only allows you to play or pause from the notification. If you want to to be able to switch between radio from the notification (I haven't implemented it in this repository) you can try implementing it with the flutter_local_notifications package

**Comment by RadioApp1** [comment]
Yeah this radio player notification is not allowing to work any custom notifications , i tried it bro , thats why I am stuck.
Do you have discord bro ? I will share my code and screen and explain in a detail way , Its the last feature i need to implement here bro , can u plss help ?

**Comment by RadioApp1** [comment]
Bro can we connect in discord ? I will share my screen and code plz
